BQE_api/glossary/urls.py

```diff
@@ -2,17 +2,18 @@
 from django.urls import path
 from django.urls.conf import include
 from .views import *
-from rest_framework import routers, serializers, viewsets
+from glossary.api.urls import *
 
-router = routers.DefaultRouter()
-router.register(r'word', WordViewSet)
+app_name = 'glossary'
```

Comment:
app_name not necessary here

Reply:
We could have this variable, if we use it to generate routes below, but probably easier to just delete it for now